.github/workflows/release-please.yml

```diff
@@ -29,6 +29,8 @@ jobs:
 
   update-docs:
```

Comment:
We want to skip this on nightly releases.

Reply:
I can add to your script below to filter nightlies

Reply:
actually, RP doesn't run on nightlies, but we still want to filter nightlies out, I'm restoring the previous script in JS because that was working